tests/e2e/smoketest-with-jaeger/03-run-check-services.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  # This is a hacky but simple way of determining if any of the traces created by tracegen were received.  It should be considered temporary.
+  - command: 'kubectl exec --namespace $NAMESPACE deployment/tempo-simplest-query-frontend --container tempo-query -- sh  -c "rm -f services ; wget -q http://localhost:16686/api/services ; cat services | grep -i tracegen"'
```

It would be better if you get the route from the Ingress. Otherwise, it'll not work with a Kubernetes cluster run in a different machine than the one used to run the tests

---

Yes, but since I'm doing a "kubectl exec" it's going to be executed in the same container, right?  And as I noted, this is just a hack for until we come up with a better solution, which I want to talk to you about shortly.

---

Oh! Good point! Sorry. I didn't notice it was done using `exec`

---

nit, but would that work too? Seems to be simpler ^^
```suggestion
  - command: 'kubectl exec --namespace $NAMESPACE deployment/tempo-simplest-query-frontend --container tempo-query -- sh  -c "curl http://localhost:16686/api/services | grep -i tracegen"'
```

---

@frzifus I don't think curl exists on those images, and whatever version of wget the have does not support -O